apps/emqx_rule_engine/src/emqx_rule_engine.appup.src

```diff
@@ -1,135 +1,130 @@
 %% -*- mode: erlang -*-
 {VSN,
-  [
-   {"4.3.6",
-    [ {update, emqx_rule_metrics, {advanced, ["4.3.6"]}}
-    , {load_module,emqx_rule_events,brutal_purge,soft_purge,[]}
-    , {load_module,emqx_rule_engine,brutal_purge,soft_purge,[]}
-    , {load_module,emqx_rule_runtime,brutal_purge,soft_purge,[]}
-    , {load_module,emqx_rule_engine_api,brutal_purge,soft_purge,[]}
-    ]},
+  [{"4.3.6",
+    [{load_module,emqx_rule_metrics,brutal_purge,soft_purge,[]},
```

Comment:
Why adding this `load_module` instruction? The `update` instruction will be translated to several low level instructions including `load_module` instruction.

Reply:
You are right, I have simply run the `scripts/update_appup.escript`.  That could be another improvement for that script (to ignore a module if there's an `update` instruction already)